src/components/Dashboard/AddressListSm.vue

```diff
@@ -9,8 +9,40 @@
         <v-expansion-panel-header id="registered-office-panel-toggle">
           <div class="list-item__title">Registered Office</div>
         </v-expansion-panel-header>
+        <!-- A completed filing is required; Don't show address information -->
```

Comment:
It's too bad about all this duplicated markup. Was it too messy to have "v-if"s at the "v-list-item-subtitle" element?

Reply:
I tried that at the beginning, but some of the elements further up had v-ifs as well as specific styling based on the coaPending. I did my best to preserve the existing behavior so I duplicated some of the mark up and removed some of the logic.